### DISCUSSION TITLE
Unclear Podfile setup docs

### DISCUSSION THREAD

**Discussion by dominictobias** [invertase/react-native-firebase#7441] [Q&A] [answered]
Hi, over here in the docs: https://rnfirebase.io/#altering-cocoapods-to-use-frameworks

This is unclear to me, it would be a lot clearer if you added to the code snippet where you meant:

> Open the file ./ios/Podfile and add this line inside your targets (right after the line calling the react native Podfile function to get the native modules config):

"Inside your targets" -> is this perhaps in this condition `target '<ProjectName>' do`

"react native Podfile function" -> is this perhaps `use_react_native!(...)`?

Thanks!

**Comment by dominictobias** [comment, marked as answer]
Actually the instructions are wrong, the lines specifically have to go BEFORE `use_react_native!(...)`. I found it by accident by following this error which mentions Firebase and others having the same issue: https://github.com/facebook/react-native/issues/36120

**Reply by mikehardy** [reply]
In fact, the instructions are correct, but only for older versions of react-native.
Unfortunately react-native has changed the way the Podfile works a couple times the most recent versions and our documentation is now stale - but I swear we try, and it did use to work.

If someone that just went through this (so they have fresh eyes!) could propose a change (there is an edit button at the top right of the page...) that separates the instructions there into a "react-native 0.71+" section with the new instructions you have, and a "react-native < 0.70" section (with current content) , that would help a lot of people
